packages/address/src/tests/AddressFormatter.test.ts

```diff
@@ -1,7 +1,7 @@
 import {fetch} from '@shopify/jest-dom-mocks';
 import {Address, FieldName} from '../types';
 import AddressFormatter from '..';
-import {mockCountryRequests} from './utilities';
+import {mockCountryRequests} from '../testUtilities';
```

Comment:
the other thing you could do is add the test utilities as a separate package.

Reply:
Love that idea 👍 
I've extracted everything in https://github.com/Shopify/quilt/pull/355 and will reuse it here once shipped published.